src/VisualStudio/Core/Def/NavigationBar/NavigationBarClient.cs

```diff
@@ -342,48 +319,17 @@ void INavigationBarPresenter.Disconnect()
             _dropdownBar.RefreshCombo((int)NavigationBarDropdownKind.Member, memberIndex);
         }
 
-        public event EventHandler<NavigationBarItemSelectedEventArgs> ItemSelected;
-
-        public event EventHandler<EventArgs> ViewFocused;
-        public event EventHandler<CaretPositionChangedEventArgs> CaretMoved;
-
-        int IVsCodeWindowEvents.OnCloseView(IVsTextView pView)
-        {
-            if (_trackedTextViews.TryGetValue(pView, out var view))
-            {
-                view.Caret.PositionChanged -= OnCaretPositionChanged;
-                view.GotAggregateFocus -= OnViewGotAggregateFocus;
```

Comment:
interesting.  what did we use focus for?

Reply:
It's used for if you switch between one view and the next -- we consider the "caret location" to be from the active view in this case.

Reply:
nevermind.  i misread.